upgrade_test.py

```diff
@@ -448,20 +448,23 @@ def test_rolling_upgrade(self):  # pylint: disable=too-many-locals,too-many-stat
                 not is_enterprise(target_upgrade_version):
             self.truncate_entries_flag = True
 
-        with self.subTest('pre-test - prepare test keyspaces and tables'):
-            # prepare test keyspaces and tables before upgrade to avoid schema change during mixed cluster.
-            self.prepare_keyspaces_and_tables()
-            self.fill_and_verify_db_data('BEFORE UPGRADE', pre_fill=True)
+        self.log.info('pre-test - prepare test keyspaces and tables')
+        # prepare test keyspaces and tables before upgrade to avoid schema change during mixed cluster.
+        self.prepare_keyspaces_and_tables()
+        self.fill_and_verify_db_data('BEFORE UPGRADE', pre_fill=True)
 
-            # write workload during entire test
-            self.log.info('Starting c-s write workload during entire test')
-            write_stress_during_entire_test = self.params.get('write_stress_during_entire_test')
-            entire_write_cs_thread_pool = self.run_stress_thread(stress_cmd=write_stress_during_entire_test)
+        # write workload during entire test
+        self.log.info('Starting c-s write workload during entire test')
+        write_stress_during_entire_test = self.params.get('write_stress_during_entire_test')
+        entire_write_cs_thread_pool = self.run_stress_thread(stress_cmd=write_stress_during_entire_test)
 
-            # Prepare keyspace and tables for truncate test
-            if self.truncate_entries_flag:
-                self.insert_rows = 10
-                self.fill_db_data_for_truncate_test(insert_rows=self.insert_rows)
+        # Let to write_stress_during_entire_test complete the schema changes
+        time.sleep(300)
```

Comment:
This is a workaround to solve the issue with the slowness of Debian9 schema agreement.
@juliayakovlev please open a new task in Trello to handle it better (wait_for_schem_agreement).

Reply:
https://trello.com/c/n6HN1SZf/2104-solve-the-issue-with-the-slowness-of-debian9-schema-agreement